internal/pkg/infrastructure/repositories/database/database.go

```diff
@@ -232,15 +232,10 @@ func seedDevices(s store, data [][]string) error {
 			types = append(types, Lwm2mType{Type: t})
 		}
 
-		var interval int = 3600
-		if interval, err = strconv.Atoi(d[11]); err != nil {
-			interval = 3600
-		}
-
 		var sensorType SensorType
 		result = s.db.First(&sensorType, "name=?", strings.ToLower(d[6]))
 		if errors.Is(result.Error, gorm.ErrRecordNotFound) {
-			newSensorType := SensorType{Name: strings.ToLower(d[6]), Interval: interval}
+			newSensorType := SensorType{Name: strings.ToLower(d[6]), Interval: 3600}
```

Comment:
Vad betyder det här? Skall vi inte kunna specificera interval längre?

Reply:
Jo, men det där är kvar för att om man inte har seedat sensortypen så kommer den att hårda in 3600 som default intervall. Sensortyp som redan är seedat kan man ställa in intervall för. Senare används intervall från devices.csv för att lägga upp sensorns intervall. Kanske kan man ta första sensorns intervall för att skapa upp en sensortyp i stället. Eller köra en panic om inte sensortypen redan finns.